### DISCUSSION TITLE
Dockerized deployment

### DISCUSSION THREAD

**Discussion by keller00** [davidmyersdev/octo#112] [Ideas]
I think it would help new users if there was a Docker image maintained by the dev team that would allow them to easily deploy Octo onto their servers and keep them updated.

For example another tool that I used has the following documentation about how to deploy it in Docker: https://traggo.net/install/#docker
Here's how they build the Docker files: https://github.com/traggo/build

**Comment by davidmyersdev** [comment]
Hey @keller00! Thanks for the suggestion. I'm definitely open to the idea of a published docker container. Octo is a static site though, so the published static assets are usually better served over a CDN or static file host such as GitHub Pages or AWS S3. The production version of Octo - [octo.app](https://octo.app) - is actually served by GitHub Pages currently.

**Comment by keller00** [comment]
Does that mean that the self hosted version doesn't have support for accounts? I think however that the one hosted on GH pages does have account support.

How does that work?

**Comment by davidmyersdev** [comment]
@keller00, apologies for the delayed reply here - I must have snoozed this notification without seeing it. The domain [octo.app](https://octo.app) is actually a little more feature rich than the open source version here. While it is almost in parity, the hosted version builds on top of this repo by integrating with Firebase to offer OAuth account management (signing in with Google or GitHub) and document syncing via Cloud Firestore.
